spec/integration/_dev/benchmark/spec.yml

```diff
@@ -6,11 +6,13 @@ spec:
       type: folder
       name: system
       required: false
-      additionalContents: false
+      additionalContents: true
       contents:
         - description: Folder containing deployable services for benchmarks.
           type: folder
           name: deploy
+          additionalContents: false
+          required: false
           $ref: "./deploy/spec.yml"
```

Comment:
In  `deploy/spec/.yml`, additionalContents and required are already defined.

```
spec:
  additionalContents: false
  required: true
```

what is the final value for `required` here?